test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -130,12 +139,7 @@ if (OSSFUZZ)
     set_target_properties(stack_reuse_codegen_ossfuzz PROPERTIES LINK_FLAGS ${LIB_FUZZING_ENGINE})
     target_compile_options(stack_reuse_codegen_ossfuzz PUBLIC
             ${COMPILE_OPTIONS}
-            -Wno-sign-conversion
-            -Wno-inconsistent-missing-destructor-override
-            -Wno-unused-parameter
-            -Wno-zero-length-array
```

Comment:
You removed `-Wno-zero-length-array` and `-Wno-unused-parameter` from the list, was that intentional? Note that the flags used here are not quite the same as in the other targets.

Reply:
I just build the `ossfuzz` and it builds without any issue. So I guess it is fine to remove those flags.